monai/handlers/tensorboard_handlers.py

```diff
@@ -258,10 +258,10 @@ def __init__(
             interval: plot content from engine.state every N epochs or every N iterations, default is 1.
             epoch_level: plot content from engine.state every N epochs or N iterations. `True` is epoch level,
                 `False` is iteration level.
-            batch_transform: a callable that is used to transform the
-                ``ignite.engine.batch`` into expected format to extract several label data.
-            output_transform: a callable that is used to transform the
-                ``ignite.engine.output`` into expected format to extract several output data.
+            batch_transform: a callable that is used to construct `[images, labels]` pair from `engine.state.batch`.
+                then plot image from `transformed[0][index]` and plot label from `transformed[1][index]`.
+            output_transform: a callable that is used to extract the `predictions` data from `ignite.engine.output`,
```

Comment:
output_transform: a callable that is used to transform the output achieved from `ignite.engine.state.output`...

Reply:
updated in the latest commit.
Thanks.